File: backend/src/main/java/br/com/restaurante/gestao_restaurante/dto/comanda/ComandaResponseDTO.java
package br.com.restaurante.gestao_restaurante.dto.comanda;

import java.time.LocalDateTime;


import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ComandaResponseDTO {
    private Long id;
    private Long mesaId;
    private Long garcomId;
    private String status;
    private String nome;
    private Double valorTotal;
    private LocalDateTime dataAbertura;
    private LocalDateTime dataFechamento;
    private boolean taxaServico;
}

File: frontend/src/dto/comanda/ComandaResponseDTO.ts
export interface ComandaResponseDTO {
    id: number;
    mesaId: number;
    garcomId: number;
    status: string;
    nome: string;
    valorTotal: number;
    dataAbertura: string;
    dataFechamento?: string;
    taxaServico: boolean;
}
